content/people/damiri.md

```diff
@@ -0,0 +1,27 @@
+name: Delaram Amiri 
```

Comment:
Please fix the formatting and indentation of the YAML metadata front
matter. It should look like that in the example file.

Also add the separator lines `---` before and after the metadata.